libs/tup-components/src/projects/ProjectsListing/ProjectSummary.tsx

```diff
@@ -15,24 +15,27 @@ export const ProjectSummary: React.FC<{
     project.allocations?.reduce((acc, e) => acc + e.used, 0) ?? 0;
 
   return (
-    <div>
-      <Link to={`/projects/${project.id}`}>
-        <span className={styles['project-title']}>{project.title}</span>
+    <>
```

Comment:
**Explanation:** Just dropping extra `<div>`s. The [React.Fragment short syntax](https://reactjs.org/docs/fragments.html#short-syntax) allows this. Extra wrappers can be helpful, but they often cause difficulty for CSS which must burrow when selecting. _Ask and I will elaborate._